rust.properties

```diff
@@ -0,0 +1,13 @@
+category=Languages
```

Comment:
This PR is incomplete. You also need to register the plugin key in sonar-update-center-properties